packages/eslint-plugin-lit-a11y/lib/rules/aria-attrs.js

```diff
@@ -4,97 +4,44 @@
 
-const validAriaProps = [
-  'aria-activedescendant',
-  'aria-atomic',
-  'aria-autocomplete',
-  'aria-busy',
-  'aria-checked',
-  'aria-controls',
-  'aria-describedat',
-  'aria-describedby',
-  'aria-disabled',
-  'aria-dropeffect',
-  'aria-expanded',
-  'aria-flowto',
-  'aria-grabbed',
-  'aria-haspopup',
-  'aria-hidden',
-  'aria-invalid',
-  'aria-label',
-  'aria-labelledby',
-  'aria-level',
-  'aria-live',
-  'aria-multiline',
-  'aria-multiselectable',
-  'aria-orientation',
-  'aria-owns',
-  'aria-posinset',
-  'aria-pressed',
-  'aria-readonly',
-  'aria-relevant',
-  'aria-required',
-  'aria-selected',
-  'aria-setsize',
-  'aria-sort',
-  'aria-valuemax',
-  'aria-valuemin',
-  'aria-valuenow',
-  'aria-valuetext',
-];
-
-module.exports = {
+/** @type {import("eslint").Rule.RuleModule} */
+const AriaAttrsRule = {
   meta: {
+    type: 'suggestion',
     docs: {
       description: 'Elements cannot use an invalid ARIA attribute.',
-      category: 'Fill me in',
+      category: 'Accessibility',
       recommended: false,
     },
-    fixable: null, // or "code" or "whitespace"
-    schema: [
-      // fill in your schema
-    ],
+    fixable: null,
+    schema: [],
   },
 
   create(context) {
-    // variables should be defined here
-
-    //----------------------------------------------------------------------
-    // Helpers
-    //----------------------------------------------------------------------
-
-    // any helper functions should go here or else delete this section
-
-    //----------------------------------------------------------------------
-    // Public
-    //----------------------------------------------------------------------
-
     return {
-      TaggedTemplateExpression: node => {
-        if (
-          node.type === 'TaggedTemplateExpression' &&
-          node.tag.type === 'Identifier' &&
-          node.tag.name === 'html'
-        ) {
+      TaggedTemplateExpression(node) {
+        if (isHtmlTaggedTemplate(node)) {
           const analyzer = TemplateAnalyzer.create(node);
 
           analyzer.traverse({
             enterElement(element) {
-              // eslint-disable-next-line
-              for (const attr in element.attribs) {
-                if (attr.startsWith('aria-')) {
-                  if (!validAriaProps.includes(attr)) {
-                    const loc = analyzer.getLocationForAttribute(element, attr);
-                    context.report({
-                      loc,
-                      message: 'Elements cannot use an invalid ARIA attribute.',
-                    });
-                  }
+              for (const attr of Object.keys(element.attribs)) {
+                if (isInvalidAriaAttribute(attr)) {
+                  const loc = analyzer.getLocationForAttribute(element, attr);
+                  context.report({
+                    loc,
+                    message: 'Invalid ARIA attribute "{{attr}}".',
```

Comment:
TIL, nice

Reply:
yup, makes it easier to refactor to use `messageId` down the line